gulpfile.js

```diff
@@ -120,7 +120,7 @@ const wpconfig = require('./webpack.pl.config');
 const serverconfig = {
   host: '0.0.0.0',
   port: '8080',
-  contentBase: path.resolve(__dirname, 'dist/', 'public/'), // ie dist/public
+  contentBase: path.resolve(__dirname, 'dist/'), // ie dist/pl
```

Comment:
`// ie dist/pl` means nothing to me. Would a reader need more context? SOCRITICALNOTREALLY

Reply:
we'll be sure to get more notes around this in the docs repo, but this should hopefully be enough in context from the webpack doc link in the comments above and the terminal output: `Listening at http://0.0.0.0:8080. PL located at http://0.0.0.0:8080/pl` what contentBase is / where pl is coming from.